pygfx/objects/_scene.py

```diff
@@ -13,6 +13,8 @@ class Background(WorldObject):
     Can be e.g. a gradient, a static image or a skybox.
     """
 
-    def __init__(self, material):
-        super().__init__()
-        self.material = material
+    def __init__(self, geometry=None, material=None):
+        # Allow one arg, as an exception
```

Comment:
this is a surprising gotcha :) why?

Reply:
I am afraid people will do `SomeBackground(SomeBackgroundMaterial())`, thus omitting the geometry, and getting weird errors. What I should do instead here, is check for this and raise an error. Will fix.